log4j-slf4j2-impl/src/main/java/org/apache/logging/slf4j/Log4jEventBuilder.java

```diff
@@ -43,6 +44,7 @@ public class Log4jEventBuilder implements LoggingEventBuilder {
     private Throwable throwable = null;
     private Map<String, String> keyValuePairs = null;
     private final Level level;
+    private String fqcnOverride;
```

Comment:
```suggestion
    private String fqcn = FQCN;
```

Reply:
This saves a conditional `fqcnOverride != null ? fqcnOverride : FQCN`, which would give `FQCN` in most cases.

Reply:
done